src/client.rs

```diff
@@ -257,6 +244,18 @@ pub struct ClientStats {
     pub indexes: HashMap<String, IndexStats>,
 }
 
+/// Health of the MeiliSearch server.
+/// Example:
+/// ```text
+/// Health {
+///    status: "available".to_string(),
```

Comment:
You used the `status` field as if it was not optional here.
I don't know where to find documentation for the upcoming meilisearch version so I was not able to check if it is really nullable/optional. But you must either remove the `Option` at line 256 either use `Some("available".to_string())` at line 251.

Reply:
It's not an `Option`, you're right! I went too fast!